src/it/java/net/spy/memcached/CancelFailureModeTest.java

```diff
@@ -101,6 +103,16 @@ public ClientMode getClientMode() {
       public FailureMode getFailureMode() {
         return FailureMode.Cancel;
       }
+
+      @Override
+      public SSLContext getSSLContext() {
```

Comment:
Why do you need to do this if this method is already implemented in the base class?

Reply:
Refactored as suggest